components/dribbble/sources/new-shot-created/new-shot-created.mjs

```diff
@@ -0,0 +1,33 @@
+import dribbble from "../../dribbble.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "dribbble-new-shot-created",
+  name: "New Shot Created",
+  description: "Emit new events when new shots are created",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    dribbble,
+    timer: {
+      type: "$.interface.timer",
+      label: "Polling schedule",
+      description: "How often to poll the Dribble API",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+  },
+  async run() {
+    const shots = await this.dribbble.getShots();
```

Comment:
I'd suggest using pagination just in case there are more than 30 shots (https://developer.dribbble.com/v2/#pagination)